src/common/file.c

```diff
@@ -104,6 +118,71 @@ device_dax_size(const char *path)
 	(void) os_close(fd);
 	errno = olderrno;
 
+	LOG(4, "device size %zu", size);
+	return size;
+}
+
+/*
+ * device_dax_align -- (internal) checks the alignment of a given dax device
+ */
+static size_t
+device_dax_align(const char *path)
```

Comment:
This name implies that the function somehow aligns the device.
`device_dax_alignment`